.github/workflows/docs-language-linter.yaml

```diff
@@ -35,6 +35,24 @@ jobs:
       - uses: errata-ai/vale-action@reviewdog
         with:
           reporter: github-pr-check
+  docs_linkcheck:
```

Comment:
Can be part of the docs lint job instead of taking another github runner (we get limited runners)

Reply:
@alonmr moved